### ISSUE TITLE
No posts showing

### ISSUE THREAD

**Issue by LoeizD** [dashdashzako/hugo-journal#5]
I installed the theme and I don't have any post showing but I see my posts with other themes. 
<img width="1298" alt="shot" src="https://user-images.githubusercontent.com/8215178/48661998-2a559780-eab6-11e8-8aa2-9685e66b5525.png">

I even tried adding that to the index.html template but still nothing. This code is copied from a website to list all pages, I'm new to Hugo.

```
{{ define "main" }}
{{.Content}}
<ul class="all-posts">
{{range .Site.Pages}}
<li><a href="{{.Permalink}}">{{.Title}}</a></li>
{{end}}
</ul>
{{ end }} 
```

Even going to the urls that work on other themes returns a 404 (like going to http://localhost:1313/posts/cool-post/)

**Comment by johnjago** [comment]
@LoeizD I had a similar issue and fixed it by updating to the latest version of Hugo. Previously I was running v0.16 (it's the version that `sudo apt install hugo` gave me), so I installed the latest binary release (v0.52) [from GitHub](https://github.com/gohugoio/hugo/releases) as suggested at the top of the install documentation. The newer version also has better error messages which might help with debugging other problems.

**Comment by ghost** [comment]
Had this issue as well. There is a very simple fix. 
Inside index.html in the template, change this line:
`    {{ range (where .Data.Pages "Type" "post").GroupByDate "2006" -}}`

You have to change "post" to "posts".
Then it works again :)

**Comment by olingern** [comment]
@peterhijma 's fix works for me on `v0.57.2`

**Comment by dashdashzako** [comment]
I admit I'm super late answering this, I apologise 🙇‍♂️

The way posts were retrieved has changed to `site.RegularPages`, and this should no longer be an issue.
